sources/packages/api/src-sql/Applications/Add-col-relationship-student-number.sql

```diff
@@ -0,0 +1,15 @@
+-- Add relationship_status
+ALTER TABLE
+    sims.applications
+ADD
+    COLUMN IF NOT EXISTS relationship_status sims.relationship_status;
```

Comment:
Just to confirm, this is not a NOT NULL column anymore? in previous commit I saw a default value of Single to it.

Reply:
Yes, i removed it as its nullable